public/javascripts/password-change.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-unused-vars */
+
+function inputCheck() {
+  if (
+    document.getElementById("old-password").value === "" ||
+    document.getElementById("new-password").value === "" ||
+    document.getElementById("repeat-password").value === ""
+  ) {
+    document.getElementById("submit-password-change").disabled = true;
```

Comment:
Repeated DOM element queries should be extracted to a constant due to performance impact of querying the same thing again multiple times.